src/utils/spo.ts

```diff
@@ -694,5 +696,27 @@ export const spo = {
     const res = await request.get<{ AadObjectId: { NameId: string, NameIdIssuer: string } }>(requestOptions);
 
     return res.AadObjectId.NameId;
+  },
+
+  async getWebRetentionLabelInformationByName(webUrl: string, name: string): Promise<ListItemRetentionLabel> {
+
```

Comment:
Same here, 
I see you're struggling with git. It might be best if you just created a new branch and copy paste the right parts of the code in there. And then create a new PR.